src/components/pages/devices/flyouts/deviceDetails/accordion.js

```diff
@@ -33,15 +33,19 @@ export class Accordion extends Component {
       <div className={joinClasses('accordion', className)}>
         <SectionHeader>
           <div className="accordion-title" onClick={this.toggleContent}>{title}</div>
-          <Btn onClick={this.toggleContent} svg={svgs.chevron} className={joinClasses("accordion-toggle-btn", this.state.showContent ? 'chevron-close' : 'chevron-open')} />
+          <button onClick={this.toggleContent} className="accordion-toggle-btn">
+            <Svg path={svgs.chevron} className={this.state.showContent ? 'chevron-close' : 'chevron-open'} />
+          </button>
         </SectionHeader>
 
-        <div className="section-content" style={displayContent}>
-          {
-            description &&
-            <SectionDesc>{description}</SectionDesc>
-          }
-          {children}
+        <div className="section-content-container">
```

Comment:
I don't see why there are two divs  here. Seems like there should just be one.

Reply:
Because I needed to be able to hide the actual contents (section-content), but also maintain the spacing (using section-content-container).